docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -90,54 +90,62 @@ The following table explains what happens at each step.
 
 <table>
   <tr>
```

Comment:
The command+description info doesn't look good in a table. Commands need the full width of the page. This table should be converted to a list.